### ISSUE TITLE
MDXGetTitle failed when trying to open some MDX files.

### ISSUE THREAD

**Issue by unspacyar** [yosshin4004/portable_mdx#8]
Hello. I found some MDX files that fails when trying to reproduce them.  I get the following error message:
**MDXGetTitle failed.**

Maybe the issue is on the MDX (broken or corrupted file maybe?).
I am including those files as attachment.

Thank you!!!

[msc004.mdx.zip](https://github.com/yosshin4004/portable_mdx/files/12388812/msc004.mdx.zip)
[msc027.mdx.zip](https://github.com/yosshin4004/portable_mdx/files/12388813/msc027.mdx.zip)
[msc028.mdx.zip](https://github.com/yosshin4004/portable_mdx/files/12388815/msc028.mdx.zip)
[msc035.mdx.zip](https://github.com/yosshin4004/portable_mdx/files/12388816/msc035.mdx.zip)
[msc036.mdx.zip](https://github.com/yosshin4004/portable_mdx/files/12388817/msc036.mdx.zip)

**Comment by Kaens** [comment]
Yes, they are quite broken in this regard! I saw them elsewhere, too, and they don't have the same format at all.

逆にこのファイルは、MDXに見えるけど何かが変・・PDXフィールドがないし、ポインタも存在してなさそうで…
何これ？どうすれば聴けます？
[unknown2.zip](https://github.com/yosshin4004/portable_mdx/files/14485853/unknown2.zip)
